tests/test_components/test_sensors/test_Sensor.py

```diff
@@ -18,8 +19,41 @@ def test_init():
     LOG.info(sensor)
 
 
-def test_get_size():
-    """Test get_size method."""
+def test_get_pitch():
+    """Test get_pitch method."""
+
+    sensor = Sensor(pitch=10 * unit.um)
+
+    result = sensor.get_pitch()
+    LOG.info(result)
+
+    assert result.unit == unit.um
+
+
+def test_get_n_px():
+    """Test get_n_px method."""
+
+    sensor = Sensor(n_px=(640, 512) * unit.pix)
+
+    result = sensor.get_n_px()
+    LOG.info(result)
+
+    assert result.unit == unit.pix
+
+
+def test_get_n_bin():
+    """Test get_n_bin method."""
+
+    sensor = Sensor(pitch=10 * unit.um, n_bin=4 * unit.dimensionless_unscaled)
+
+    result = sensor.get_n_bin()
+    LOG.info(result)
+
+    assert result.unit == unit.dimensionless_unscaled
+
+
+def test_get_shape() -> tuple:
```

Comment:
No need for return type here since test functions only return pass/fail.
```suggestion
def test_get_shape():
```